contracts/lbp/LBPManager.sol

```diff
@@ -86,37 +89,39 @@ contract LBPManager {
         uint256[] memory _startWeights,
         uint256[] memory _startTimeEndTime,
         uint256[] memory _endWeights,
-        uint256 _swapFeePercentage,
-        uint256 _feePercentage
+        uint256[] memory _fees,
+        bytes memory _metadata
```

Comment:
I'm wondering if something like this actually needs to be stored on the blockchain, or if it wouldn't be sufficient to emit this as an event parameter. That way if someone wants to check the parameters he'd need to check the events for this contract, there would be just one "initialization" event, that would contain the metadata. The data doesn't actually fulfill any role onchain, right? (with that I mean it will never be accessed for any function or state transition). Wonder how you think about that, @kinspotting

Reply:
yes I agree. It would be good to do. But some talks were going on, that in future they might want to update the project details through admin page. But for now I think we can opt out from storing it on chain.

Reply:
Let's move this to a new PR. Change that needs to be implemented:- `Don't store metadata. Instead, just emit it which can be used by the front end.`.